docs/build/Build-for-ARTIK053-TizenRT.md

```diff
@@ -28,68 +38,56 @@ cd iotjs-tizenrt
 git clone https://github.com/Samsung/iotjs.git
 git clone https://github.com/Samsung/TizenRT.git
 ```
-The following directory structure is created after these commands
 
+The following directory structure is created after these commands
 ```bash
 iotjs-tizenrt
   + iotjs
-  |  + config
-  |      + tizenrt
-  |          + artik05x
   + TizenRT
 ```
 
-#### 2. Add IoT.js as a builtin application for TizenRT
+#### 2. Configure IoT.js
+
+This step is optional. You can edit the build config of IoT.js in `iotjs/build.config`.
+For example, edit `iotjs-include-module` to add module as below.
 
-```bash
-cp iotjs/config/tizenrt/artik05x/app/ TizenRT/apps/system/iotjs -r
-cp iotjs/config/tizenrt/artik05x/configs/ TizenRT/build/configs/artik053/iotjs -r
-cp iotjs/config/tizenrt/artik05x/romfs.patch TizenRT/
+```
+"iotjs-include-module": ["adc", "gpio"]
 ```
 
 #### 3. Configure TizenRT
 
 ```bash
 cd TizenRT/os/tools
 ./configure.sh artik053/iotjs
+cd ..
 ```
 
-#### 4. Configure ROMFS of TizenRT
+#### 4. Copy your app in ROMFS
 
 ```bash
-cd ../../
-patch -p0 < romfs.patch
-cd build/output/
-mkdir res
-# You can add files in res folder
-# The res folder is later flashing into the target's /rom folder
+cp <your test app> ../tools/fs/contents/
+# make ROMFS image
+../tools/fs/mkromfsimg.sh
+# The contents folder is later flashing into the target's /rom folder
 ```
 
+NOTE: The romfs image file must not exceed 400kb.
+
 #### 5. Build IoT.js for TizenRT
 
 ```bash
-cd ../../os
-make context
-cd ../../iotjs
-./tools/build.py --target-arch=arm --target-os=tizenrt --sysroot=../TizenRT/os --target-board=artik05x --clean
+make IOTJS_ROOT_DIR=../../iotjs
 ```
 
+If there is no `IOTJS_ROOT_DIR` argument, [IoT.js](https://github.com/Samsung/TizenRT/tree/master/external/iotjs) released in TizenRT is built, not `iotjs-tizenrt/iotjs`.
```

Comment:
My suggestion is like below:

If `IOTJS_ROOT_DIR` isn't given as the argument, IoT.JS codes, which is located in `external/iotjs` of TizenRT, will be used for the build.